packages/toolkit/src/components/LatestChangesCard.tsx

```diff
@@ -0,0 +1,79 @@
+"use client";
+import { useEffect, useState } from "react";
+import axios from "axios";
+import { Icons } from "@instill-ai/design-system";
+
+interface Changelog {
+  id: string;
+  createdAt: string;
+  updatedAt: string;
+  date: string | null;
+  title: string;
+  published: boolean;
+}
+
+const LatestChangesCard: React.FC = () => {
+  const [changelogs, setChangelogs] = useState<Changelog[]>([]);
+
+  useEffect(() => {
+    axios
+      .get(
+        "https://productlane.com/api/v1/changelogs/52f06d0d-2381-411e-a8c5-7b375e3a0114"
+      )
+      .then((response) => {
+        const data: Changelog[] = response.data;
+        const filteredAndSortedChangelogs = data
+          .filter((changelog) => changelog.published)
+          .sort(
+            (a, b) =>
+              new Date(b.createdAt).getTime() - new Date(a.createdAt).getTime()
+          );
+        setChangelogs(filteredAndSortedChangelogs.slice(0, 3));
+      })
+      .catch((error) => {
+        console.error("Error fetching changelogs:", error);
+      });
+  }, []);
```

Comment:
@thewbuk This will make the console request this URL every re-render which is not good.

Please re-write this quest under a react-query useQuery function and utilize the quest hook to get the data, this case the data will be cached